packages/Pager/src/pager.scss

```diff
@@ -0,0 +1,174 @@
+@use '~@igloo-ui/tokens/dist/base10/variables' as tokens;
+@use '~@igloo-ui/tokens/dist/fonts';
+
+:root {
+  /* Default */
+  --ids-pager-font-family: #{tokens.$primary-font-family};
+  --ids-pager-font-size: #{tokens.$font-size-4};
+  --ids-pager-line-height: #{tokens.$line-height-xxxl};
+
+  /* Item */
+  --ids-pager-item-padding: #{tokens.$space-1};
+  --ids-pager-item-background: #{tokens.$samoyed};
+  --ids-pager-item-border: 1px solid #{tokens.$grey-300};
+  --ids-pager-item-prev-border-radius: #{tokens.$border-radius-sm} 0 0 #{tokens.$border-radius-sm};
+  --ids-pager-item-next-border-radius: 0 #{tokens.$border-radius-sm} #{tokens.$border-radius-sm} 0;
+  --ids-pager-item-color: #{tokens.$grey-800};
+
+  /* Selected Item */
+  --ids-pager-item-selected-background: #{tokens.$electric-blue-500};
+  --ids-pager-item-selected-color: #{tokens.$samoyed};
+
+  /* Hover */
+  --ids-pager-item-hover-border: 1px solid #{tokens.$grey-600};
+
+  /* Focus */
+  --ids-pager-item-focus-border: 1px solid #{tokens.$electric-blue-600};
+
+  /* Disabled */
+  --ids-pager-item-disabled-background: #{tokens.$grey-300};
+  --ids-pager-item-disabled-border: 1px solid #{tokens.$grey-300};
+  --ids-pager-item-disabled-color: #{tokens.$grey-600};
+
+  /* Arrows */
+  --ids-pager-arrow-color: #{tokens.$grey-500};
+
+  /* Ellipsis */
+  --ids-pager-ellipsis-color: #{tokens.$grey-600};
+
+  /* Results */
+  --ids-pager-results-color: #{tokens.$grey-600};
+  --ids-pager-results-margin: 0 0 0 #{tokens.$space-3};
+}
+
+%focus-style {
+  border: var(--ids-pager-item-focus-border);
+}
+
+%hover-style {
+  border: var(--ids-pager-item-hover-border);
+}
+
+.ids-pager {
+  align-items: center;
+  display: flex;
+  font-family: var(--ids-pager-font-family);
+  font-size: var(--ids-pager-font-size);
+  line-height: var(--ids-pager-line-height);
+
+  &,
+  & * {
+    box-sizing: border-box;
+  }
+}
+
+.ids-pager__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.ids-pager__item {
+  display: flex;
+  flex: 0 0 auto;
+}
+
+.ids-pager__button {
```

Comment:
The pager rendering on safari does not match the design. 

**Safari**
<img width="457" alt="pager-safari" src="https://user-images.githubusercontent.com/2379339/196212888-2cd04b5b-84cf-450d-968d-f1e2b7054620.png">

to fix the rendering it is necessary to add these two statements:
```
appearance: none;
margin: 0;
```